File: SoloLearn/Code Coach/Izzy the Iguana/IzzyIguana.py
snacks = input().split()
total = 0
for snack in snacks:
    if snack == "Lettuce":
        total += 5
    elif snack == "Carrot":
        total += 4
    elif snack == "Mango":
        total += 9
if total > 9:
    print("Come on Down!")
else:
    print("Time to wait")

File: SoloLearn/Code Coach/Izzy the Iguana/IzzyIguana.rb
snacks = gets.chomp.split
total = 0
for snack in snacks
    case snack
    when "Lettuce"
        total += 5
    when "Carrot"
        total += 4
    when "Mango"
        total += 9
    end
end
if total > 9
    puts "Come on Down!"
else
    puts "Time to wait"
end